java/src/Ice/src/main/java/com/zeroc/IceInternal/Instance.java

```diff
@@ -1880,4 +1894,16 @@ private NetworkProxy createNetworkProxy(com.zeroc.Ice.Properties properties, int
     private static boolean _oneOffDone = false;
     private QueueExecutorService _queueExecutorService;
     private QueueExecutor _queueExecutor;
+
+    private Map<String,String[]> _builtInModulePackages = java.util.Collections.unmodifiableMap(new HashMap<String, String[]>() {{
```

Comment:
Is "package" accurate here?

As far as I can tell, the associated name is not a package but a prefix. For example, the Glacier2 package is com.zeroc.Glacier2, not com.zeroc.

Reply:
PackagePrefix would be more correct.